keps/sig-node/606-compute-device-assignment/kep.yaml

```diff
@@ -6,7 +6,7 @@ authors:
   - "@renaudwastaken"
 owning-sig: sig-node
 participating-sigs: []
-status: implemented
+status: implementable
```

Comment:
why going back to implementeable? It's implemented, no?

Reply:
because work is still needed on a KEP which is basically done :) asked advice on sig-arch: https://groups.google.com/g/kubernetes-sig-architecture/c/v4OwOaBkvVc/m/xdfLryLTGQAJ